persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/AggregateEventTag.scala

```diff
@@ -3,10 +3,9 @@
  */
 package com.lightbend.lagom.javadsl.persistence
 
-import scala.annotation.tailrec
-import akka.persistence.journal.WriteEventAdapter
-import akka.persistence.journal.WriteEventAdapter
-import akka.persistence.journal.Tagged
+import org.pcollections.{ PSequence, TreePVector }
+
+import scala.collection.JavaConverters._
 
 object AggregateEventTag {
```

Comment:
Sorry for joining the party late, but would it make sense to deprecate the `of` method and rename it `singleton`. I think it's too easy for users to default to `of` without understanding the implications. Also, I think the documentation of the `of` method needs to be updated to explain the implications of using `of` versus `shard`.